NGTrackForce/src/app/components/associate-list-page/associate-list-page.component.ts

```diff
@@ -72,6 +79,34 @@ export class AssociateListPageComponent implements OnInit, OnDestroy, AfterViewI
      */
 
     // Add option for none
+    const thingy2 = JSON.parse(this.lsHelp.localStorageItem("currentUser"));
+    if (thingy2.role === 2){
```

Comment:
i dont understand what is going on in this block of code

Reply:
checks if user is a trainer and if so, use a different associate service, refactored